int-tests/src/test-integration/java/io/neow3j/protocol/Neow3jWriteIntegrationTest.java

```diff
@@ -212,4 +216,43 @@ public void testSendToAddress_TransactionSendAsset() throws IOException {
         assertNotNull(tx);
     }
 
+    @Test
+    public void cancelTransactionFailsBecauseTransactionWasAlreadyConfirmed() throws IOException {
+        TransactionSendToken param = new TransactionSendToken(IntegrationTestHelper.NEO_HASH, BigInteger.ONE,
+                defaultAccountAddress());
+        Transaction tx = getNeow3j().sendToAddress(param).send().getSendToAddress();
+
+        waitUntilTransactionIsExecuted(tx.getHash(), getNeow3j());
+
+        NeoCancelTransaction cancel = getNeow3j()
+                .cancelTransaction(tx.getHash(), asList(Hash160.fromAddress(defaultAccountAddress())), null)
+                .send();
+
+        assertThat(cancel.getError().getMessage(), is("This tx is already confirmed, can't be cancelled."));
+    }
+
+    /*
+     * This test can have timing issues. The transaction is sent and then immediately cancelled. But, if the
+     * transaction is included in a block before the cancellation is processed, the test will fail.
+     */
```

Comment:
Is there any way to increase the block time for the NeoTestContainer that is started for this test (and this test only, i.e., not for any other test in this file)?

Reply:
No, we set the block time in the config.json in our `test-tools` module.